app/helpers/notification_count.rb

```diff
@@ -2,12 +2,17 @@ module ExercismWeb
   module Helpers
     module NotificationCount
       def notification_count
```

Comment:
Yes, if we could make it so that it ONLY adjusts specific things that we ask for, and not anything else, then it would be worth it.